### ISSUE TITLE
 AttributeError: module 'pypandoc' has no attribute 'convert'

### ISSUE THREAD

**Issue by kelsos** [ethereum/eth-keyfile#36]
* Version: 0.6.0 (any version)
* Python: 3.9
* OS: osx 12.4 (apple silicon)

### What was wrong?

A recent release of pypandoc removed deprecated code including the `convert` method.

`setuptools-markdown` has been deprecated and has no update since 2019, and has no pinned version of pypandoc so it pulls the new version of pypandoc which results in the following error. 

```bash
(eth-keyfile) ➜  eth-keyfile git:(master) pip install -e .
Obtaining file:///Users/kelsos/development/eth-keyfile
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [22 lines of output]
      /Users/kelsos/.virtualenvs/eth-keyfile/lib/python3.9/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
        warnings.warn(
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/Users/kelsos/development/eth-keyfile/setup.py", line 46, in <module>
          setup(
        File "/Users/kelsos/.virtualenvs/eth-keyfile/lib/python3.9/site-packages/setuptools/__init__.py", line 87, in setup
          return distutils.core.setup(**attrs)
        File "/Users/kelsos/.virtualenvs/eth-keyfile/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 109, in setup
          _setup_distribution = dist = klass(attrs)
        File "/Users/kelsos/.virtualenvs/eth-keyfile/lib/python3.9/site-packages/setuptools/dist.py", line 477, in __init__
          _Distribution.__init__(
        File "/Users/kelsos/.virtualenvs/eth-keyfile/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 293, in __init__
          self.finalize_options()
        File "/Users/kelsos/.virtualenvs/eth-keyfile/lib/python3.9/site-packages/setuptools/dist.py", line 901, in finalize_options
          ep(self)
        File "/Users/kelsos/.virtualenvs/eth-keyfile/lib/python3.9/site-packages/setuptools/dist.py", line 922, in _finalize_setup_keywords
          ep.load()(self, ep.name, value)
        File "/Users/kelsos/development/eth-keyfile/.eggs/setuptools_markdown-0.4.1-py3.9.egg/setuptools_markdown.py", line 43, in long_description_markdown_filename
          output = pypandoc.convert(markdown_filename, 'rst', format='md')
      AttributeError: module 'pypandoc' has no attribute 'convert'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

### How can it be fixed?

Probably replacing setuptools-markdown with something else?

**Comment by radrow** [comment]
Can we have a new tag so package managers can work with it?

**Comment by kclowes** [comment]
Closed via #37. @radrow I will plan to release in the next day or two.
